unsupported/f5_plugins/apic_iapp/iapps/hitesh.tmpl

```diff
@@ -238,7 +238,7 @@ proc get_var { name } {
 # Input: $name = name of variable    
 #        $checkvalue = the string that disables the option
 #        $option = TMSH name of the option
-#        $module = the BIG-IP module that enables the option
+#        $module = the BIG-IP®module that enables the option
```

Comment:
Hmm is it possible to remove this? I'm not sure what will happen if we have this in an iApp template. This is technically code.